src/main/app/forms/form.ts

```diff
@@ -77,6 +80,23 @@ export class Form<Model> {
       .map((error: FormValidationError) => error.message)[0]
   }
 
+  /**
+   * Get raw data for given field name.
+   * @param {string} fieldName
+   * @returns {object}
+   */
+  rawDataFor (fieldName: string): object {
```

Comment:
The logic inside this method seems to be more complex than just returning a field value. Can please you give some more info on what this method should be doing? Can you please add some unit tests for it?

Reply:
The method does exactly the same thing as `valueFor` just taking data from `rawData` property. Could be refactored a bit and must be tested 👍